packages/storybook/stories/components/toggle/migration.from.17.x.mdx

```diff
@@ -0,0 +1,87 @@
+import { Meta } from '@storybook/blocks';
+import * as ToggleStories from './toggle.stories';
+
+<Meta of={ ToggleStories } name='Migration From 17.x' />
+
+# Toggle - migrate from v17 to v18
+----
+
+## Attributes changes
+
+`checked` <img src="https://img.shields.io/badge/updated-00FFFF" />
+
+Has been updated.
+
+You can use the new `value` attribute to obtain the same behavior.
+
+`checking` <img src="https://img.shields.io/badge/removed-FF0000" />
+
+Has been removed.
+
+Design team doesn't valid this style anymore.
+
+`contrasted` <img src="https://img.shields.io/badge/removed-FF0000" />
+
+Has been removed.
+
+Design team doesn't valid this style anymore.
+
+`color` <img src="https://img.shields.io/badge/removed-FF0000" />
+
+Has been removed.
+
+Design team doesn't valid this style anymore.
+
+`disabled` <img src="https://img.shields.io/badge/updated-00FFFF" />
+
+Has been updated.
+
+You can use the new `isDisabled` attribute to obtain the same behavior.
+
+`interactive` <img src="https://img.shields.io/badge/removed-FF0000" />
+
+Has been removed.
+
+Toggle is now always interactive.
+
+`isRequired` <img src="https://img.shields.io/badge/new-008000" />
+
+New attribute (optional).
+
+Defines if the toggle is required in a form.
+
+`name` <img src="https://img.shields.io/badge/new-008000" />
+
+New attribute.
+
+You can use the new name attribute for set an identifiant in a form.
+
+`withLabel` <img src="https://img.shields.io/badge/new-008000" />
+
+New attribute (optional).
+
+For add a label `ON` / `OFF`
+
+## Migration examples
+
+Checked toggle:
+```html
+<osds-toggle checked>
+</osds-toggle>
+
+<!-- is now -->
+
+<ods-toggle value>
```

Comment:
this looks really weird (though it will looks better on React with value={ true })